### ISSUE TITLE
Authentication and Authorization

### ISSUE THREAD

**Issue by rodrigorm** [evoluhq/evolu#219]
Hey, does Evolu have any plans to add authentication and authorization features? I'm brainstorming an app that uses local-first architecture. The idea is to let users share data and assign roles and permissions for how they can interact with that data. What do you think?

**Comment by steida** [comment]
@rodrigorm Yes, but it will differ from what you can see in other collaboration libraries. The reason is privacy and no third-party dependency. It will be based on asymmetric cryptography. I have a PoC and it should be done this year.

**Comment by steida** [comment]
For now, use whatever third-party service that is suitable for your needs. Evolu will bring its own solution soon.

**Comment by MirjamElad** [comment]
>  @tylermercer : For now I can just create a separate Evolu instance and share the mnemonic for that instance between team members

How is that working out for you?

> @steida : For now, use whatever third-party service that is suitable for your needs. Evolu will bring its own solution soon.

Any idea when collaboration would be possible/implemented in evolu?

**Comment by steida** [comment]
@MirjamElad Two months max.

**Comment by tylermercer** [comment]
> @MirjamElad : How is that working out for you?

I actually switched to Dexie. As cool as Evolu is, I found it difficult to integrate with SvelteKit. I ultimately realized that CRDTs and E2EE aren't necessary for the simple, lovable, and complete version of what I'm building. Dexie has been much easier to use so far.

(@steida I'm definitely going to keep watching Evolu though! I think it has a lot of potential. Just a bit too new for my project, and I don't need all its features.)

**Comment by steida** [comment]
@tylermercer Thank you for your honest (and kind) response. I'm just committing an update for easier Svelte support.

**Comment by steida** [comment]
Consider https://www.npmjs.com/package/@lo-fi/webauthn-local-client

**Comment by mesterum** [comment]
> For now, use whatever third-party service that is suitable for your needs. Evolu will bring its own solution soon.

How soon?
What about a [decentralized authentication and authorization for team collaboration, using a secure chain of cryptographic signatures.](https://github.com/local-first-web/auth)? I am curious about your solution.
